flink-examples/src/main/java/io/pravega/examples/flink/Utils.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.examples.flink;
+
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.stream.Stream;
+import io.pravega.client.stream.StreamConfiguration;
+import io.pravega.connectors.flink.PravegaConfig;
+
+public class Utils {
+
+    /**
+     * Creates a Pravega stream with a default configuration.
+     *
+     * @param pravegaConfig the Pravega configuration.
+     * @param streamName the stream name (qualified or unqualified).
+     */
+    public static Stream createStream(PravegaConfig pravegaConfig, String streamName) {
+        return createStream(pravegaConfig, streamName, StreamConfiguration.builder().build());
+    }
+
+    /**
+     * Creates a Pravega stream with a given configuration.
+     *
+     * @param pravegaConfig the Pravega configuration.
+     * @param streamName the stream name (qualified or unqualified).
+     * @param streamConfig the stream configuration (scaling policy, retention policy).
+     */
+    public static Stream createStream(PravegaConfig pravegaConfig, String streamName, StreamConfiguration streamConfig) {

```

Comment:
Since `StreamConfiguration` can be derived from `StreamConfiguration.builder().build()`, is there any reason for accepting this as a parameter?